sky/serve/serve_state.py

```diff
@@ -34,7 +34,6 @@ def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
         status TEXT,
         uptime INTEGER DEFAULT NULL,
         policy TEXT DEFAULT NULL,
-        auto_restart INTEGER DEFAULT NULL,
```

Comment:
We cannot remove this column since this will introduce inconsistency between users using the new/old version of SkyServe. IIUC, we should keep this column but never access it, just like in `spot_state` (just add a comment to say this column is deprecated): https://github.com/skypilot-org/skypilot/blob/7c514baabe1f2452d19354bfa804923e45755afd/sky/spot/spot_state.py#L33-L58

cc @Michaelvll for a double-check.

Reply:
sounds good. Added